### DISCUSSION TITLE
Почистить проблемы путём закрытия неактуальных и переноса части в обсуждения

### DISCUSSION THREAD

**Discussion by rominf** [rominf/latex-g7-32#133] [Ideas]
Большое количество незакрытых проблем может отпугивать пользователей.

Предлагаю:
1. Закрыть неактуальные.
2. Перенести всё, что не является реальными проблемами, которые можно исправить в обозримом будущем, в обсуждения.

Что думаете?

**Comment by afaikiac** [comment]
Часть уже можно перенести в G7-32
